src/test/resources/speedtest/travelSearch.csv

```diff
@@ -1,2 +1,3 @@
 testCaseId,description,departure,arrival,window,origin,fromPlace,fromLat,fromLon,destination,toPlace,toLat,toLon,category,modes
 1,Window-1h20m-NE-to-SW,12:00,14:00,1h20m,,,45.53381,-122.64344,,,45.49511,-122.69723,transit,TRANSIT
+2,SingleIteration-NE-to-SW,12:00,14:00,1m,,,45.53381,-122.64344,,,45.49511,-122.69723,transit,TRANSIT
```

Comment:
Should't search window be 0, as per https://github.com/opentripplanner/OpenTripPlanner/blob/dev-2.x/src/main/java/org/opentripplanner/raptor/api/request/SearchParams.java#L139

Reply:
Yes it should, but for some reason it does not work, so I had to set it to 1m.